lua/nvim-treesitter/highlight.lua

```diff
@@ -33,6 +33,7 @@ end
 function M.start(bufnr, lang)
   local parser = parsers.get_parser(bufnr, lang)
   ts.highlighter.new(parser, {})
+  vim.treesitter.start(bufnr, lang)
```

Comment:
```suggestion
  vim.treesitter.start(bufnr, lang)
```

Reply:
In fact, that whole function (and `stop()`) can go; just `vim.treesitter.start()` in `attach()` (and similarly for `detach()`).